carbonmark/components/pages/Project/Purchase/Pool/PurchaseModal.tsx

```diff
@@ -16,6 +16,10 @@ export interface Props {
     value: string;
     token: CarbonmarkToken;
   };
+  approvalValue?: {
```

Comment:
this purchase modal will always be USDC, so I believe `token` can be removed and approvalValue can just be a value